index.html

```diff
@@ -415,21 +415,21 @@ <h3>
           "https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
           with a time value equal to the time immediately after the <a>current
           document</a>'s <a data-cite=
-          "!HTML/parsing.html#the-end">DOMContentLoaded event</a> completes.
+          "HTML/parsing.html#the-end">DOMContentLoaded event</a> completes.
```

Comment:
```suggestion
          "HTML/parsing.html#event-domcontentloaded">DOMContentLoaded</a> event completes.
```

Reply:
As I noted above, you might want to define this as a dependency, but you need to define it like this:

```HTML
<code><dfn data-cite="HTML/parsing.html#event-domcontentloaded">DOMContentLoaded<dfn></code>
```

Then just do:

```
<a>DOMContentLoaded</a> event...
```

And ReSpec will "code" it automatically for you.